client/src/apis/user.js

```diff
@@ -51,7 +52,7 @@ export const signup = async (user) => {
     const res = await axios.post("/api/users", user);
     setToken(res.data.token);
     const socket = await importSocket();
-    socket.emit("log in", _id);
+    socket.emit("log in", user._id);
```

Comment:
user doesn't have _id properties.

```suggestion
    socket.emit("log in", res.data.user._id);
```